src/pages/Vote/styled.tsx

```diff
@@ -1,29 +1,25 @@
 import styled from 'styled-components'
+import { ProposalState } from '../../state/governance/hooks'
 
-const handleColorType = (status?: any, theme?: any) => {
+const handleColorType = (status?: ProposalState, theme?: any) => {
   switch (status) {
-    case 'pending':
+    case ProposalState.Pending:
+    case ProposalState.Active:
       return theme.blue1
-    case 'active':
-      return theme.blue1
-    case 'succeeded':
+    case ProposalState.Succeeded:
+    case ProposalState.Executed:
       return theme.green1
-    case 'defeated':
+    case ProposalState.Defeated:
       return theme.red1
-    case 'queued':
-      return theme.text3
-    case 'executed':
-      return theme.green1
-    case 'canceled':
-      return theme.text3
-    case 'expired':
-      return theme.text3
+    case ProposalState.Queued:
```

Comment:
can't you just remove these cases and fall into the default?

Reply:
yeah, the explicit handling was merely meant for clarity